clients/client/src/main/java/org/projectnessie/client/http/HttpClientBuilder.java

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.client.http;
+
+import static org.projectnessie.client.NessieConfigConstants.CONF_NESSIE_AUTH_TYPE;
+import static org.projectnessie.client.NessieConfigConstants.CONF_NESSIE_OWNER;
+import static org.projectnessie.client.NessieConfigConstants.CONF_NESSIE_PASSWORD;
+import static org.projectnessie.client.NessieConfigConstants.CONF_NESSIE_REPOSITORY;
```

Comment:
can we just do a straight remove of all the repo/owner stuff. Nothing downstream is using it. I think it just adds confusion here.

Reply:
Can do it in an immediate follow-up.